css/style.css

```diff
@@ -0,0 +1,187 @@
+* {
+  box-sizing: border-box;
+}
+
+h1, h2, p, body, img, div {
```

Comment:
Love this use of DRY, gathering multiple elements in one { }

Reply:
Thanks!